client/pages/index/index.js

```diff
@@ -18,6 +18,14 @@ Page({
       url: '../editntc/editntc'
     })
   },
+  /**
+   * Jump to userlist page
+   */
+  JumpToUserslist: function () {
```

Comment:
nit: the name should be either users or user list, not users list. and the "L" in "list" should be capitalized.